R/pkg/R/DataFrame.R

```diff
@@ -1166,26 +1166,33 @@ setMethod("take",
             collect(limited)
           })
 
-#' Head
-#'
-#' Return the first \code{num} rows of a SparkDataFrame as a R data.frame. If \code{num} is not
-#' specified, then head() returns the first 6 rows as with R data.frame.
-#'
-#' @param x a SparkDataFrame.
+#' Return the first part of a SparkDataFrame or Column
+#' 
+#' If \code{x} is a SparkDataFrame, its first 
+#' rows will be returned as a data.frame. If the dataset is a \code{Column}, its first 
+#' elements will be returned as a vector. The number of elements to be returned
+#' is given by parameter \code{num}. Default value for \code{num} is 6.
+#' @param x a SparkDataFrame or Column
 #' @param num the number of rows to return. Default is 6.
 #' @return A data.frame.
 #'
 #' @family SparkDataFrame functions
 #' @aliases head,SparkDataFrame-method
-#' @rdname head
 #' @name head
 #' @export
 #' @examples
 #'\dontrun{
+#' # Initialize Spark context and SQL context
 #' sparkR.session()
-#' path <- "path/to/file.json"
-#' df <- read.json(path)
-#' head(df)
+#' 
+#' # Create a DataFrame from the Iris dataset
+#' irisDF <- as.DataFrame(iris)
```

Comment:
you might want to avoid using `iris` in example, because it will cause a warning with the column name having `.` and if at one point we fix this to support `.` in column name, the example will need to be updated